best-practices/README.md

```diff
@@ -104,11 +104,18 @@ Rails
 Testing
 -------
 
+(See also the rspec style guide that appears in this section)
+
+* First #describe what you are doing.
+* Then establish the #context.
+* #it only expects one thing (within reason).
+* Prefer explicitness - #it #its #specify #subject are nice shortcuts but
+consider whether they are sacrificing readability.
 * Avoid `any_instance` in rspec-mocks and mocha. Prefer [dependency injection].
-* Avoid `its`, `let`, `let!`, `specify`, `before`, and `subject` in RSpec.
 * Avoid using instance variables in tests.
 * Disable real HTTP requests to external services with
-  `WebMock.disable_net_connect!`.
+`WebMock.disable_net_connect!`. Use vcr if you need to create a fixture with real-life
+requests and responses from an external service.
 * Don't test private methods.
```

Comment:
`if, when you are done, your method is private, don't commit any tests for it`

@novemberkilo the guide should at least say this. A noob who blindly follows `Don't test private methods.` could needlessly make life harder for themselves.

OTOH, if you needed those specs to efficiently write those methods, chances are when you (or someone else) have to fix something or make a major change to the method you (or that other person) will wish you had them again. i.e. I still think I'm right :wink: 

You could always tag any private methods you found useful as `@private` and exclude them from the build, but still run them by default in dev. Or you could make it its own class. 

But definitely the wording above over an unexplained "don't".